backend/src/src-schedule/src/main/java/org/eclipse/sw360/schedule/timer/ScheduleConstants.java

```diff
@@ -69,8 +72,29 @@ private ScheduleConstants(){}
             invalidConfiguredServices.add(ThriftClients.CVESEARCH_SERVICE);
         }
 
+        if(! props.containsKey(DELETE_ATTACHMENT_OFFSET_PROPERTY_NAME)){
+            log.info("Property "+ DELETE_ATTACHMENT_OFFSET_PROPERTY_NAME + " not set. Using default value.");
```

Comment:
Maybe log could be `debug` since it check for a property and uses default value if not present.
Indentation `"Property "+`
Same for Line 86-87